### PR TITLE
Admin improvements

### LINKED ISSUE #60: Documentation: Add more details on the settings page.
For the moment the settings page is really light, maybe adding some details about the fields, what they actually mean can be nice...

### PR DESCRIPTION
**Pull request by macbookandrew**
Fixes #60

### CODE REVIEW

**Comment by shulard** [comment]
Hello @macbookandrew and thank you for your contribution !

Before merging that PR, can you fix the build ? It seems that you use the `__` function which must be replace by an escape version.

You can find details about the corresponding functions here : https://codex.wordpress.org/Data_Validation

You can also check locally that everything is fine using the command : 

```bash
vendor/bin/phpcs -n --standard=phpcs.xml
```

If you need help, just ask :wink:.

**Comment by macbookandrew** [comment]
@shulard Thanks; updated my code and fixed a few other issues that phpcs found.

File: options.php [hunk 1]
```diff
@@ -124,7 +124,11 @@ function wpmautic_base_url() {
  * Define the input field for Mautic script location
  */
 function wpmautic_script_location() {
-	$position = wpmautic_option( 'script_location', '' );
+	$position     = wpmautic_option( 'script_location', '' );
+	$allowed_tags = array(
+		'br'   => [],
```
Comment (shulard) [inline options.php]: Here, it's better to use the `array()` syntax because we are compatible with very old PHP versions...

**Comment by shulard** [comment]
Sorry about the delay, can you also rebase on master to retrieve the latests changes ? It must fix the Travis build...

**Pull request merged**